pkg/v1/cluster/requests_opts.go

```diff
@@ -38,4 +38,8 @@ type CreateOpts struct {
 	// EnableAutorepair reflects if worker nodes are allowed to be reinstalled automatically
 	// in case of their unavailability or unhealthiness. Enabled by default.
 	EnableAutorepair *bool `json:"enable_autorepair,omitempty"`
+
+	// EnablePatchVersionAutoUpgrade specifies if Kubernetes version of the cluster is allowed to be upgraded
```

Comment:
Maybe it's worth to mention that we are talking about patch version of k8s exactly here, e.x: 
`EnablePatchVersionAutoUpgrade  specifies if Kubernetes patch version of the cluster is allowed to be upgraded`.